src/System.Management.Automation/engine/Modules/AnalysisCache.cs

```diff
@@ -634,14 +634,14 @@ public void QueueSerialization()
                     // Wait a while before assuming we've finished the updates,
                     // writing the cache out in a timely matter isn't too important
                     // now anyway.
-                    await Task.Delay(10000);
+                    await Task.Delay(10000).ConfigureAwait(false);
```

Comment:
It is not clear to me that this is needed.  Isn't the Task always configured as "false" be default?

Reply:
It seems right though, so I think it's good to keep it.

Reply:
@PaulHigin As far as I know, the default option is `'true'`.